### PR TITLE
Firestore

### PR DESCRIPTION
**Pull request by miriamhaenle**
Hi all! This pull request includes saving the relevant data in the backend (firestore) instead of LocalStorage). Thanks in advance for the review.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -34,37 +34,40 @@ export default function App() {
...
-    const historyFootprintPerTransportationType = getFromStorage(
-      APP_STORAGE_KEYS.footprintPerTransportType
-    )
-
-    setTotalCarbonFootprint(historicTotalCarbonFootprint)
-    setFootprintPerTransportationType(
-      historyFootprintPerTransportationType || []
-    )
-  }, [])
+    if (user) {
```
Comment (web-devil) [inline src/App.js:37]: please move the updateState and save method to an own service

File: src/components/utils/handleStorage.js [hunk 2]
```diff
@@ -1,7 +1,30 @@
-export function saveToStorage(key, dataSet) {
-  localStorage.setItem(key, JSON.stringify(dataSet))
+import firebase from 'firebase'
```
Comment (web-devil) [inline src/components/utils/handleStorage.js:1]: I wouldnt recommend a 'utils' folder. 
better is a 'services' folder ... utils is to generic and broad.

File: src/App.js (hunk 1 above)
Comment (miriamhaenle) [reply src/App.js:37]: I tried that previously but for some reason I did not manage to update the third state. My approach was to write helper functions to update each state and give that to the service. For 2 of these functions it worked but for the third it always threw an error "is not a function". Thus, I left it in the App.js to make it finally work. 
If you have any suggestions for a better aproach I'd be happy to know 😄

Comment (web-devil) [reply src/App.js:40]: hmm, maybe it is a option for code review later

**Pull request merged**